OpenBSD/openbsd_packet_filter/tests/test_ingest_ipv6_carp_logs.json

```diff
@@ -0,0 +1,78 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "OpenBSD Packet Filter",
+        "dialect_uuid": "8510051d-c7cf-4b0c-a398-031afe91faa0"
+      }
+    },
+    "message": "123,001,anchor1,label2,eth0,match,pass,in,6,,123,64,12345,0,DF,112,6,80,2001:0db8:85a3:0000:0000:8a2e:0370:7334,2001:0db8:85a3:0000:0000:ac1f:0001:0023,carp,64,1,2,3,4"
+  },
+  "expected": {
+    "message": "123,001,anchor1,label2,eth0,match,pass,in,6,,123,64,12345,0,DF,112,6,80,2001:0db8:85a3:0000:0000:8a2e:0370:7334,2001:0db8:85a3:0000:0000:ac1f:0001:0023,carp,64,1,2,3,4",
```

Comment:
I'm not sure about this event. Except IPv6 address, the event seems different from the one with IPv4 address

ipv4:
```
rulenr,subrulenr,anchor,label,intf,match,pass,in,4,0x0,,63,18292,0,DF,112,vrrp,72,1.2.3.4,5.6.7.8,3,255,13,2,0,1
```

ipv6:
```
rulenr,subrulenr,anchor,label,intf,match,pass,in,6,,123,64,12345,0,DF,112,6,80,2001:0db8:85a3:0000:0000:8a2e:0370:7334,2001:0db8:85a3:0000:0000:ac1f:0001:0023,carp,64,1,2,3,4"
```

from ipv4 -> ipv6:
- `vrrp` protocol name become 6
- the value of the `carp type` 3 become `carp`